src/Tickets/Site_Health/Subsections/Features/Tickets_Commerce_Subsection.php

```diff
@@ -0,0 +1,262 @@
+					'Tickets Commerce Currency Code',
+					'event-tickets'
+				),
+				'value'    => $this->get_tickets_commerce_currency_code(),
+				'priority' => 290,
+			],
+			[
+				'id'       => 'tickets_commerce_currency_position',
+				'title'    => esc_html__(
+					'Tickets Commerce Currency Position',
+					'event-tickets'
+				),
+				'value'    => $this->get_tickets_commerce_currency_position(),
+				'priority' => 300,
+			],
+			[
+				'id'       => 'tickets_commerce_decimal_separator',
+				'title'    => esc_html__(
+					'Tickets Commerce Decimal Separator',
+					'event-tickets'
+				),
+				'value'    => $this->get_tickets_commerce_decimal_separator(),
+				'priority' => 310,
+			],
+			[
+				'id'       => 'tickets_commerce_thousands_separator',
+				'title'    => esc_html__(
+					'Tickets Commerce Thousands Separator',
+					'event-tickets'
+				),
+				'value'    => $this->get_tickets_commerce_thousands_separator(),
+				'priority' => 320,
+			],
+			[
+				'id'       => 'tickets_commerce_number_of_decimals',
+				'title'    => esc_html__(
+					'Tickets Commerce Number of Decimals',
+					'event-tickets'
+				),
+				'value'    => $this->get_tickets_commerce_number_of_decimals(),
+				'priority' => 330,
+			],
+		];
+	}
+
+	/**
+	 * Calculates the average order total for tickets commerce.
+	 *
+	 * @return string Formatted average price.
+	 * */
+	private function get_tickets_commerce_average_order_total(): string {
+		$tickets_commerce_ticket_prices = tribe( Tickets_Repository::class )->per_page( -1 )->pluck( 'price' );
+		$total                          = 0;
+		$count                          = 0;
+
+		foreach ( $tickets_commerce_ticket_prices as $price ) {
+			if ( $price === 'Free' || $price === '' || $price === null ) {
+				// Skip free or empty prices for average calculation.
+				continue;
+			} else {
+				// Match the number with international currency format.
+				preg_match(
+					'/\d+([,.]\d+)?/',
+					$price,
+					$matches
+				);
+				if ( isset( $matches[0] ) ) {
+					// Convert to a standard number format (replace comma with period).
+					$number = floatval(
+						str_replace(
+							',',
+							'.',
+							$matches[0]
+						)
+					);
+					$total  += $number;
+
+					++$count;
+				}
+			}
+		}
+
+		// Calculate the average price, avoid division by zero.
+		$tickets_commerce_average_price = $count > 0 ? $total / $count : 0;
+
+		// Format the average price with two decimal points.
+		$tickets_commerce_formatted_average_price = number_format(
+			$tickets_commerce_average_price,
+			2,
+			'.',
+			','
+		);
+
+		return $tickets_commerce_formatted_average_price;
+	}
+
+	/**
+	 * Determines if Tickets Commerce is in test mode.
+	 *
+	 * @return string 'True' if Tickets Commerce is in test mode, 'False' otherwise.
+	 */
+	private function is_tickets_commerce_test_mode(): string {
+		return $this->get_boolean_string(
+			tribe_get_option(
+				'tickets-commerce-test-mode',
+				false
+			)
```

Comment:
Did a quick codebase search and could find `tickets-commerce-test-mode` as an option. I did find `tickets-commerce-sandbox`, though. Should this be changed?
```suggestion
			tribe_get_option(
				'tickets-commerce-sandbox',
				false
			)
```

Reply:
Also, is there a reason we're not using the variable for all these options (example: `TEC\Tickets\Commerce\Settings::$option_sandbox`)?